driver/src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -41,51 +47,75 @@
  * it only provides immutable fields and has no non-fatal side effects.
  */
 public final class Opt {
-  public static final String autofuzz = stringSetting("autofuzz", "");
-  public static final List<String> autofuzzIgnore = stringListSetting("autofuzz_ignore", ',');
-  public static final String coverageDump = stringSetting("coverage_dump", "");
-  public static final String coverageReport = stringSetting("coverage_report", "");
-  public static final List<String> customHookIncludes = stringListSetting("custom_hook_includes");
-  public static final List<String> customHookExcludes = stringListSetting("custom_hook_excludes");
-  public static final List<String> customHooks = stringListSetting("custom_hooks");
-  public static final List<String> disabledHooks = stringListSetting("disabled_hooks");
-  public static final String dumpClassesDir = stringSetting("dump_classes_dir", "");
-  public static final boolean hooks = boolSetting("hooks", true);
-  public static final String idSyncFile = stringSetting("id_sync_file", null);
+  // All supported arguments are added to this set by the individual *Setting methods.
+  private static final Map<String, OptDetails> knownArgs = new TreeMap<>();
```

Comment:
Hide `OptDetails` and the different type methods in a dedicated class?

Reply:
Done, I refactored out all the parsing logic.